### ISSUE TITLE
Heroku not Using Unicorn

### ISSUE THREAD

**Issue by pablocrivella** [pablocrivella/bar-brothers-bo#22]
```
###### WARNING:
       No Procfile detected, using the default web server (webrick)
       https://devcenter.heroku.com/articles/ruby-default-web-server
```

**Comment by pablocrivella** [comment]
Heroku recommends Puma
https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server

**Comment by pablocrivella** [comment]
Fixed on release 2.0.2, switched to Puma.
